packages/protocol/contracts/common/UsingPrecompiles.sol

```diff
@@ -234,4 +234,21 @@ contract UsingPrecompiles {
     }
     return x;
   }
+
+  /**
+   * @notice Returns the minimum number of required signers for a given block number.
+   * @dev Computed in celo-blockchain as int(math.Ceil(float64(2*valSet.Size()) / 3))
+   */
+  function minQuorumSize(uint256 blockNumber) internal view returns (uint256) {
+    return numberValidatorsInSet(blockNumber).mul(2).add(2).div(3);
+  }
+
+  /**
+   * @notice Computes byzantine quorum from current validator set size
+   * @return Byzantine quorum of validators.
+   */
+  function byzantineQuorumValidatorsInCurrentSet() public view returns (uint256) {
```

Comment:
Could you rename one of these two to be match whether `min` or `byzantine` is used at the start?